extension/src/prototypes/ui-components/OverlayPopper.tsx

```diff
@@ -93,6 +95,12 @@ export const OverlayPopper: FC<OverlayPopperProps> = ({
     }
   }, [onClose, pinned]);
 
+  useEffect(() => {
+    return document
+      .getElementsByClassName(VIEWER_DOM_CLASS)?.[0]
```

Comment:
Could you make this function in [here](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/blob/9082df965b20f0d1ea6f2e67aae40f1f56b62cd6/extension/src/shared/reearth/utils/canvas.ts#L1)? 🙏

Reply:
Oh i didn't notice that. I can use that getCesiumCanvas instead. Thank you!